rgd/celery.py

```diff
@@ -11,6 +11,7 @@
 # Using a string config_source means the worker doesn't have to serialize
 # the configuration object to child processes.
 app = Celery('rgd', config_source='django.conf:settings', namespace='CELERY')
+app.conf.task_default_queue = 'default'
```

Comment:
All the other Celery config is done via Django settings. I think you should set that there, for consistency.

Reply:
With `--exclude-queues` I don't think this queue is needed.